src/Curiosity/Runtime.hs

```diff
@@ -460,7 +472,7 @@ createUserFull db newProfile = if username `elem` User.usernameBlocklist
     case mprofile of
       Just profile -> existsErr
       Nothing      -> do
-        modifyUsers db (newProfile :)
+        modifyUsers db (++ [newProfile])
```

Comment:
Do we need to add the new profile to the end of the list? I think 
```suggestion
        modifyUsers db (newProfile:)
```

Possibly reads better and might be more efficient.

Reply:
Indeed the code would be slightly nicer and appenddind is worse than consing performance-wise but my goal is "user-friendlyness" when looking at the state of the system, one of the major goal of the prototype. (This was done on purpose here: https://github.com/hypered/curiosity/pull/53/commits/e44c0c8b1925b13598ce562746d4558c2df075e3)